google/cloud/forseti/notifier/notifier.py

```diff
@@ -77,25 +76,26 @@ def find_pipelines(pipeline_name):
         LOGGER.error('Can\'t import pipeline %s: %s', pipeline_name, e.message)
 # pylint: enable=inconsistent-return-statements
 
-def _get_timestamp(global_configs, statuses=('SUCCESS', 'PARTIAL_SUCCESS')):
+def _get_latest_inventory_index_id(service_config):
     """Get latest snapshot timestamp.
 
     Args:
-        global_configs (dict): Global configurations.
-        statuses (tuple): Snapshot statues.
+        service_config (dict): 2.0 service configs.
 
     Returns:
         string: The latest snapshot timestamp.
     """
 
-    latest_timestamp = None
-    try:
-        current_dao = dao.Dao(global_configs)
-        latest_timestamp = current_dao.get_latest_snapshot_timestamp(statuses)
-    except db_errors.MySQLError as err:
-        LOGGER.error('Error getting latest snapshot timestamp: %s', err)
-
-    return latest_timestamp
+    allowed_status = ('SUCCESS', 'PARTIAL_SUCCESS')
+    inventory_index_id = ''
+    with service_config.scoped_session() as session:
+        for item in DataAccess.list(session):
+            if item.status in allowed_status:
+                inventory_index_id = item.id
```

Comment:
Do we need to break out of the loop once we found the inventory_index_id?

Reply:
No, because we are looking for the last record here, so the loop must continue to the end.  This is not my preferred way, but sqlalchemy doesn't support finding the last record, and I don't see an easy way to sort the datetime which is a string.

Reply:
I tested the string sort for the datetime stamp and it works.  So, following up with PR #1070 to use sort, rather than the loop.